src/test/java/org/openmrs/performance/utils/CommonUtils.java

```diff
@@ -33,4 +35,11 @@ public static List<String> extractConceptIds(String response) {
 		return conceptIds;
 	}
 
+	public static String getDate(int daysToAdjust) {
+		// Get current date-time and adjust by the specified number of days
+		ZonedDateTime adjustedDateTime = ZonedDateTime.now().plusDays(daysToAdjust);
+
+		return adjustedDateTime.format(DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSXXX"));
+	}
```

Comment:
Split responsibilities into clearly named, single-purpose functions. It makes the code cleaner.
```suggestion
    public static String getCurrentDateTimeAsString() {
        ZonedDateTime now = ZonedDateTime.now();
        return formatDateTime(now);
    }

    public static String getAdjustedDateTimeAsString(int daysToAdjust) {
        ZonedDateTime adjustedDateTime = ZonedDateTime.now().plusDays(daysToAdjust);
        return formatDateTime(adjustedDateTime);
    }

    private static String formatDateTime(ZonedDateTime dateTime) {
        return dateTime.format(DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSXXX"));
    }
```

Reply:
also write unit tests for above functions in `src/test/java/org/openmrs/performance/utils/TestCommonUtils.java` file. 

Opening a separate PR for the util would be better I guess.

Reply:
Sure @jayasanka-sack , will open a new PR for this